apps/meteor/ee/app/message-read-receipt/server/index.js

```diff
@@ -0,0 +1,5 @@
+import { onLicense } from '../../license/server';
+
+onLicense('message-read-receipt', () => {
+	require('./hooks');
```

Comment:
You can use `await import('....')` to not use require 👀

Reply:
Done! Also converted this file to TS